plug.go

```diff
@@ -12,6 +12,7 @@ type Plug struct {
 	ViewsRemaining int
 	Approved       bool
 	PresignedURL   string
+    Shape          string
```

Comment:
Is github just rendering this indentation wrong or is it less indented than everything else? gofmt is your friend.